tests/units/units.cpp

```diff
@@ -915,7 +915,7 @@ TEST(Units, checkingOwningModelBothUnits)
 
     model->addUnits(u1);
     model->addUnits(u2);
-    model->addUnits(u3); // Should fail. No base units defined in model
+    model->addUnits(u3); // u not defined in model
```

Comment:
No need for such comments. (Same for the one below).

Reply:
Will make those changes now.